spec/puppetserver/utils/http_client_spec.rb

```diff
@@ -53,7 +51,10 @@
       FileUtils.cp(settings[:cacert], settings[:localcacert])
       FileUtils.cp(settings[:cacrl], settings[:hostcrl])
 
-      store = subject.make_store(settings[:localcacert], :chain, settings[:hostcrl])
+      client = Puppetserver::Utils::HttpClient.new(settings[:localcacert],
+                                                      :chain,
```

Comment:
indentation is off here, should be in vertical line with the start of `settings` above.